sdks/framework/typescript/steps/inbox.mdx

```diff
@@ -1,13 +1,26 @@
 ---
 title: "In-App Channel Step"
 sidebarTitle: "Inbox"
+icon: "inbox"
 ---
 
+The `inbox` step allows you to send a message to your Inbox component. This is useful when you need to send a message to a user's inbox, such as a mobile app or web app.
+
 <RequestExample>
     ```typescript Inbox
     await step.inApp('inbox', async () => {
         return {
-            body: 'A new notification has been created',
+          subject: 'Welcome to Acme!',
+          body: 'We are excited to have you on board.',
+          avatar: 'https://acme.com/avatar.png',
+          primaryAction: {
```

Comment:
@rifont shall we wait for a new inbox component for this PR to be merged?

Reply:
We don't need to wait, this change support both the old NC and Invox